reana_client/cli/workflow.py

```diff
@@ -757,8 +784,16 @@ def workflow_delete(ctx, workflow, all_runs, workspace,
 @add_access_token_options
 @click.pass_context
 def workflow_diff(ctx, workflow_a, workflow_b, brief,
-                  access_token, context_lines):
-    """Show diff between two worklows."""
+                  access_token, context_lines):  # noqa: D301
+    """Show diff between two worklows.
+
+    The `diff` command permits to compare two workflows. Workflow_a and
```

Comment:
The `diff` command permits to compare two workflows, the workflow_a and workflow_b, which must be provided as arguments. The output will show the difference in workflow run parameters, the generated files, the logs, etc.